src/Database/PostgreSQL/Consumers/Consumer.hs

```diff
@@ -27,18 +28,19 @@ instance ToSQL ConsumerID where
 
 instance Show ConsumerID where
   showsPrec p (ConsumerID n) = showsPrec p n
-
+ 
 -- | Register consumer in the consumers table,
 -- so that it can reserve jobs using acquired ID.
 registerConsumer
```

Comment:
Interface change will require a major version bump, so let's bump the version number now and mention this change in the changelog.

Reply:
I tried hard to preserve the current interface and only add 1 new function. There are no functionality changes only additions. According to https://pvp.haskell.org/, this should mean bumping minor version (2.0.0.2 -> 2.0.1.0).

Reply:
You added a new type class constraint, which can cause compile failure in code compatible with `2.0.*`. So we'll have to bump the version to `2.1`.

Reply:
Ah, now I get it. Ok. Done.